src/store/modules/shipment/mutations.ts

```diff
@@ -8,6 +8,13 @@ const mutations: MutationTree <ShipmentState> = {
   },
   [types.SHIPMENT_CURRENT_UPDATED] (state, payload) {
     state.current = payload.current;
-  }
+  },
+  [types.SHIPMENT_UPDATE_PRODUCT_COUNT] (state, id) {
+    state.current.items.forEach((item: any) => {
+      if(item.productId === id.payload){
```

Comment:
I think we should improve `id.payload`. Pass this value directly as payload or payload should be object with id

Reply:
Made the requested changes.